app/Http/Controllers/Auth/LoginController.php

```diff
@@ -37,4 +40,13 @@ public function __construct()
     {
         $this->middleware('guest')->except('logout');
     }
+
+    protected function redirectTo()
+    {
+        if (Auth()->user()->role_id == config('auth.roles.admin')) {
+            return route('admin.');
```

Comment:
```suggestion
            return route('admin');
```
chỗ này thừa dấu . phải ko e?

Reply:
không chị chỗ này là em để name admin thế cho dễ phân biệt ạ